Intersect (Core)/Config/Options.cs

```diff
@@ -127,6 +127,8 @@ public class Options
 
         public static int MaxChatLength => Instance.ChatOpts.MaxChatLength;
 
+        public static bool LogChatMessages => Instance.ChatOpts.LogChatMessages;
```

Comment:
I believe we are moving away from top level options and are instead moving them to categories.

Ie reference the chatopts instance , rather than the underlying options.